charts/ingress/templates/ingress.yaml

```diff
@@ -1,60 +1,66 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: ingestion
+  name: ingress
   annotations:
     kubernetes.io/ingress.class: azure/application-gateway
     appgw.ingress.kubernetes.io/backend-path-prefix: "/"
+    cert-manager.io/cluster-issuer: letsencrypt-prod
 spec:
+  tls:
+    - hosts:
+        - {{ .Values.ingressHostname }}
+      secretName: phdi-playground-issuer-account-key
   rules:
-    - http:
+    - host: {{ .Values.ingressHostname }}
+      http:
         paths:
           - path: /ingestion/*
             pathType: Prefix
             backend:
               service:
-                name: phdi-{env}-ingestion-service
+                name: ingestion-service
                 port:
                   number: 8080
           - path: /alerts/*
             pathType: Prefix
             backend:
               service:
-                name: phdi-{env}-alerts-service
+                name: alerts-service
```

Comment:
Also not needed for this PR, but I'm a little curious about the behavior of the ingress controller when not all the services are present. If we're not running record linkage for instance, what happens when traffic is routed to the `/record-linkage` endpoint?

Reply:
Basically, it just throws a `502 Bad Gateway`, and will spit out an error in the event logs for the ingress in Azure